CHANGELOG.md

```diff
@@ -4,18 +4,33 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased](https://github.com/ruflin/Elastica/compare/8.x...9.x)
+### Backward Compatibility Breaks
+* `Elastica\IndexTemplate` now works only with new `_index_template` API [#2258](https://github.com/ruflin/Elastica/pull/2258)
+* Remove `useLegacy` argument from `Elastica\IndexTemplate` ; `Elastica\IndexTemplate` now can only work with new `_index_template` API [#2258](https://github.com/ruflin/Elastica/pull/2258)
+### Added
+### Changed
+### Deprecated
+### Removed
+### Fixed
+### Security
+
 ## [Unreleased](https://github.com/ruflin/Elastica/compare/8.1.0...8.x)
```

Comment:
@sidz Looks like we missed to update this here as the changes below are against 9.x?

Reply:
@ruflin I assume we will change this before we release 9.0